utils/req-res-log-valiadtor.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python3
+import os
+import glob
+import json
+import jsonschema
+import subprocess
+import redis
+import time
+import argparse
+import multiprocessing
+import collections
+import io
+import signal
+import traceback
+from datetime import timedelta
+from functools import partial
+try:
+    from jsonschema import Draft201909Validator as schema_validator
+except ImportError:
+    from jsonschema import Draft7Validator as schema_validator
+
+"""
+The purpose of this file is to validate the reply_schema values of COMMAND DOCS.
```

Comment:
let's list all the validations this tool currently does in some comment in the code, and also add a list of additional validations we're considering in the future (i know they are in a todo bullet in the PR, but let's list them here as well)

Reply:
so far, the script will fail only if one of the replies doesn't comply with its schema. 
we would also like the script to fail if:
1. one of the commands in COMMANDS DOCS doesn't have the reply_schema field (can't do that for now, we want CI to pass even if some schemas are missing, because we want ti to verify the schemas we do have)
2. the testsuite didn't execute all of the commands (but that includes cluster commands, which are covered in runtest-cluster, we will need to run them both before running the validator)
3. one or more of the branches of the reply schema (e.g. oneOf, anyOf) was not hit. not sure how to do that, we will need to think of something.

let's keep that GH comment unresolved, and handle it when we are near the end (i.e. all commands have a reply_schema + we can run and validate runtest-cluster)

Reply:
note that i think the comment can list the both current and future validations (and an indication of what's implement).
also, some validations can result in a warning print (without failing the execution), we can change from warning to error later on.